File: 	1047.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');


let [horai, minutoi, horaf, minutof] = lines.shift().split(" ");
let ht= horaf - horai;
let mt = minutof - minutoi;

 if(ht === 0 && mt===0){
  ht += 24;
    
 }
 if(mt < 0){
  ht--;
 mt += 60;

 }
 if(ht <0){
    ht += 24;
 }
 console.log(`O JOGO DUROU ${ht} HORA(S) E ${mt} MINUTO(S)`)

File: 	1047.py
horai=0
minutoi=0
horaf=0
minutof=0

horai, minutoi, horaf, minutof = input().split(" ")
horai= int(horai)
minutoi= int(minutoi)
horaf= int(horaf)
minutof= int(minutof)

ht= horaf - horai
mt = minutof - minutoi

if ht == 0 and mt == 0:
  ht += 24

if mt < 0:
 ht-=1
 mt += 60

if ht <0:
  ht += 24
    
print('O JOGO DUROU {} HORA(S) E {} MINUTO(S)'.format(ht, mt))
 
